client/blocks/login/index.jsx

```diff
@@ -80,6 +86,10 @@ class Login extends Component {
 	render() {
 		return (
 			<div>
+				<div className="login__form-header">
+					{ this.props.twoStepNonce ? this.props.translate( '2-Step Verification' ) : this.props.title }
```

Comment:
It is unclear to me why `Login` has a `title` prop. We should probably just remove it altogether:

```js
{ twoFactorAuthType ? translate( '2-Step Verification' ) : translate( 'Log in to your account.' ) }
```